### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by kkmanuu**
✔️ I Defines a reusable Coin component for displaying cryptocurrency details.

✔️Implements a Navbar component for navigation and icons in the web application.

✔️Configures a Redux store with reducers for coins, global state, and single coin data.

✔️Creates Redux actions and reducers to fetch and manage individual coin data.

✔️Sets up Redux actions and reducers for fetching and managing global cryptocurrency data.

✔️Implements Redux actions and reducers for fetching and managing a list of coins.

✔️Provides Jest tests for fetching and processing individual coin data, handling errors.

✔️I Contain a Jest test to ensure accurate retrieval of a list of cryptocurrency coins.

 ✔️I Include a Jest test to verify correct global cryptocurrency data retrieval.

✔️I Define a function to fetch detailed data for an individual cryptocurrency.

✔️Implements a function to fetch and process a list of cryptocurrency coins with basic details.

✔️I Defines a function to fetch and process global cryptocurrency data.

✔️I Defines the main App component with routing for Home and Details pages.

✔️I Implement the Home page component displaying a list of cryptocurrency coins and links.

✔️I Renders the main App component using ReactDOM's createRoot function.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,142 @@
...
+If you like this project give start.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :handshake: Contributing <a name="contributing"></a>
+Contributions, issues [page issues](https://github.com/kkmanuu/math-magicians/issues), and feature requests are welcome!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+I would like to thank  Microverse for this opportunity.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (LionRouge1) [inline README.md]: - [x] Kindly add in your "Acknowledgements" section an acknowledgement that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @kkmanuu, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
almost

## Highlights
- The project is a single page application (SPA) using `react-router-dom` :heavy_check_mark: 
- Recorded a video presenting the features of the project :heavy_check_mark: 
- Write unit tests for components :heavy_check_mark: 
- The project contains a `Home` page and a `Detail` page. :heavy_check_mark: 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (kkmanuu) [reply README.md]: @LionRouge1 Done making the required changes.

**Review by cynthiainga — APPROVED** [review]
Hi @kkmanuu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] The project use `react-router-dom` ✔️ 
- [ ] There are `Home` and `Details` pages ✔️ 
- [ ] Add requested change ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**